circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   node:
-      version: 4.2.1
+      version: 8.4.0
```

Comment:
It might be better to keep supporting v4 since v8 is not LTS version yet. Here is the schedule:
https://github.com/nodejs/Release#release-schedule1

Reply:
Thanks for sharing the information. In 4.2.x, some dependencies could not be resolved. 
But I'll try to upgrade dependencies too and support 4.x.